src/services/ScheduleService.ts

```diff
@@ -119,10 +120,20 @@ const createDailyForecast = async () => {
 
   const sunData = (await axios.get(sunUrl+sunQueryParams)).data.response.body.items.item;
   const fcstData = (await axios.get(fcstUrl+fcstQueryParams)).data.response.body.items.item;
-  const wrnData = (await axios.get(wrnUrl+wrnQueryParams)).data.response;
+  const wrnRes = (await axios.get(wrnUrl+wrnQueryParams)).data.response;
+
+  const wrnArr = [8, 5, 7, 2, 3, 12, 1, 6, 9, 4]
+  let wrnCode = null
+
+  if (wrnRes.header.resultCode!='03') {
+      const wrnData = wrnRes.body.items.item;
+
+      const indexArr = wrnData.map((element) => {
+          return wrnArr.indexOf(element.warnVar);
+      })
 
-  if (wrnData.header.resultCode!='03') {
-      console.log(3456)
+      const minIndex = (indexArr.indexOf(Math.min(...indexArr)));
+      wrnCode = wrnData[indexArr[minIndex]].warnVar;
```

Comment:
```suggestion
      wrnCode = wrnArr[minIndex];
```
제가 잘 이해한 건지 모르겠지만 이렇게 해도 같은 의미가 맞을까요..?